p2p/src/peer_manager/helpers.rs

```diff
@@ -61,3 +60,27 @@ where
 
     (address, peer_info1, peer_info2)
 }
+
+/// Return first event that is accepted by predicate.
+///
+/// Used to skip events that are not of interest or that are different between backends
+/// (for example ConnectivityEvent::Discovered).
+/// Can be used in tests only, will panic in case of errors.
+pub async fn filter_connectivity_event<T, F>(
```

Comment:
Can we move this function or even the whole `peer_manager/helpers.rs` module to `testing_utils.rs`? Can we at least rename the helpers module to "test helpers" or something like that.

Reply:
Good idea. Moved out all to `testing_utils.rs`.